doc/source/whatsnew/v0.24.0.rst

```diff
@@ -27,6 +27,7 @@ New features
 - :meth:`DataFrame.corr` and :meth:`Series.corr` now accept a callable for generic calculation methods of correlation, e.g. histogram intersection (:issue:`22684`)
 - :func:`DataFrame.to_string` now accepts ``decimal`` as an argument, allowing the user to specify which decimal separator should be used in the output. (:issue:`23614`)
 - :func:`DataFrame.read_feather` now accepts ``columns`` as an argument, allowing the user to specify which columns should be read. (:issue:`24025`)
+- :func:`DataFrame.to_html` now accepts ``render_links`` as an argument, allowing the user to generate HTML with links to any URLs that appear in the DataFrame. (:issue:`2679`)
```

Comment:
can you add a ref to the new docs that you added.

Reply:
Need a link here or just a mention of the addition to docs?

Reply:
If you add a link I can approve and merge

Reply:
@jreback @WillAyd Link added and all green.